code/go/0chain.net/smartcontract/dbs/event/event.go

```diff
@@ -108,11 +60,50 @@ func (edb *EventDb) GetEvents(block int64) ([]Event, error) {
 }
 
 func (edb *EventDb) drop() error {
-	return edb.Store.Get().Migrator().DropTable(&Event{})
+	err := edb.Store.Get().Migrator().DropTable(&Event{})
+	if err != nil {
+		return err
+	}
+
+	err = edb.Store.Get().Migrator().DropTable(&Blobber{})
+	if err != nil {
```

Comment:
Can we use error wrapping here? It makes traces easy to trace https://pkg.go.dev/github.com/pkg/errors#Wrap

Reply:
We are not using https://pkg.go.dev/github.com/pkg. This method is only being used in tests so just passing up the error looks fine. Maybe I should more the method to a `_test.go` file.